ml/src/main/scala/ai/h2o/sparkling/ml/algos/H2OGridSearch.scala

```diff
@@ -364,12 +359,22 @@ class H2OGridSearch(override val uid: String) extends Estimator[H2OMOJOModel]
   }
 
   override def copy(extra: ParamMap): this.type = defaultCopy(extra)
+
+  private def extractH2OParameters(algo: H2OSupervisedAlgorithm[_, _, _ <: Model.Parameters]): Model.Parameters = {
+    val m = algo.getClass.getDeclaredMethod("updateH2OParams")
+    m.setAccessible(true)
+    m.invoke(algo)
+
+    val field = PojoUtils.getFieldEvenInherited(algo, "parameters")
```

Comment:
Isn't a concept of `POJO` deprecated?

Reply:
This is something a bit different -> PojoUtils means `Plain Old Java Object Utils` and are some sort H2O's reflection utils. This particular call just finds the `parameters` field, even if it's defined in any of the super classes

Reply:
Ok, thanks for your explanation!